integrations/github/integration.py

```diff
@@ -1,15 +1,29 @@
+from typing import Literal
 from pydantic import Field
 from port_ocean.core.handlers.port_app_config.models import (
     PortAppConfig,
     ResourceConfig,
+    Selector,
 )
 from port_ocean.core.handlers.port_app_config.api import APIPortAppConfig
 from port_ocean.core.integrations.base import BaseIntegration
 
 
+class GithubPullRequestSelector(Selector):
+    state: str = Field(
+        default="open",
+        description="Filter by pull request state (e.g., open, closed, all)",
+    )
```

Comment:
lets state options

Reply:
and in the github app atm we are also supporting querying back to a specific time period, lets make sure we are supporting similar capabilities